metis-media-service/src/main/java/eu/europeana/metis/mediaservice/EdmObject.java

```diff
@@ -0,0 +1,130 @@
+package eu.europeana.metis.mediaservice;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Function;
+
+import org.jibx.runtime.BindingDirectory;
+import org.jibx.runtime.IBindingFactory;
+import org.jibx.runtime.IMarshallingContext;
+import org.jibx.runtime.IUnmarshallingContext;
+import org.jibx.runtime.JiBXException;
+
+import eu.europeana.corelib.definitions.jibx.Aggregation;
+import eu.europeana.corelib.definitions.jibx.HasView;
+import eu.europeana.corelib.definitions.jibx.RDF;
+import eu.europeana.corelib.definitions.jibx.WebResourceType;
+
+public class EdmObject implements Serializable {
+	
+	private static IBindingFactory rdfBindingFactory;
+	
+	static {
+		try {
+			rdfBindingFactory = BindingDirectory.getFactory(RDF.class);
+		} catch (JiBXException e) {
+			throw new RuntimeException("Unable to create binding factory", e);
+		}
+	}
+	
+	private final RDF rdf;
```

Comment:
Since `RDF` is not serializable, I need to look up and use other serialization methods with Storm. This renders the serialVersionUID discussion irrelevant :)